src/fides/api/models/privacy_preference_v2.py

```diff
@@ -206,10 +216,10 @@ def privacy_experience_config_history_id(cls) -> Column:
         )
 
     # The specific experience under which the user was presented the relevant notice
-    # Minimal information stored here, mostly just region and component type
-    @declared_attr
-    def privacy_experience_id(cls) -> Column:
-        return Column(String, ForeignKey("privacyexperience.id"), index=True)
+    # Minimal information stored here, mostly just region
+    privacy_experience_id = Column(
+        String
+    )  # TODO will be removed.  Preemptively removing the FK to Privacy Experience
```

Comment:
This is used on both the Privacy Preference History and Served Notice History columns - it had a FK to privacy_experience. However, now adding and removing a region will add/remove the Privacy Experience.  We will no longer store this for preferences and notices served. The same details are on the Privacy Experience Config History Id and the user geography.